buildSrc/src/main/resources/META-INF/gradle-plugins/dev.icerock.mobile.multiplatform-resources.properties

```diff
@@ -0,0 +1 @@
+implementation-class=dev.icerock.gradle.MultiplatformResourcesPlugin
```

Comment:
move plugin to separated gradle-module with publication to maven, update sample to usage maven version of plugin